Userland/Utilities/dd.cpp

```diff
@@ -36,6 +40,29 @@ enum Status {
     Noxfer
 };
 
+Status status = Default;
+size_t total_bytes_copied = 0;
+size_t total_blocks_in = 0, partial_blocks_in = 0;
+size_t total_blocks_out = 0, partial_blocks_out = 0;
+Core::ElapsedTimer timer { true };
+
+static void closing_statistics()
+{
+    if (status != None) {
+        warnln("{}+{} blocks in", total_blocks_in, partial_blocks_in);
+        warnln("{}+{} blocks out", total_blocks_out, partial_blocks_out);
+        if (status != Noxfer) {
+            auto elapsed_time = timer.elapsed_time();
+            DeprecatedString copy_speed = "INF B/s";
```

Comment:
i guess that wording it as “instantaneous” (or “very/sooo quick” in other words) could be also nice; thatʼs something more to my taste, at least. 😅